handlers/README.md

```diff
@@ -0,0 +1,58 @@
+# Handlers
+
+Handlers in Kiali are associated with a particular Route defined in [routes.go](../routing/routes.go). When a request matches the URL pattern for a Route, the associated Handler is called. For example, when the router receives a request at `/api/clusters/health` the `ClustersHealth` handler is then called.
+
+Handlers must conform to the [net/http HandlerFunc](https://pkg.go.dev/net/http#HandlerFunc) signature. They typically create [business service(s)](../business/) _per request_ but they also need access to a number of objects that are initialized only once when the Kiali server first starts and then are reused across requests like the [config object](../config/config.go) or the [kiali cache](../kubernetes/cache/cache.go). In order to pass these cross request dependencies, handlers should be defined as closures, allowing the handler to be called once with the cross request dependencies when it is first defined in [routes.go](../routing/routes.go), and then invoked again repeatedly as a `HandlerFunc` when it is matched in the router.
```

Comment:
Just to confirm, `config` never changes in a Kiali pod's lifetime, right?  And is safe for use in a closure?